content/2015.09.kinto-encryption-example.rst

```diff
@@ -0,0 +1,207 @@
+How to encrypt data with Kinto.js
+###########################################
+
+:date: 2015-09-16
+:summary: How to encrypt data with Kinto.js.
+:slug: kinto-encryption-example
+:lang: en
+:url: kinto-encryption-example
+
+A guest post this time, by Michiel de Jong from the Firefox OS team! Sorry for
+writing in English. :)
```

Comment:
You do not have to be sorry for the english version since it will eventually be translated in French at some point.